src/tests/ftest/launch.py

```diff
@@ -2186,47 +2192,47 @@ def process(self, test, repeat, stop_daos, archive, rename, jenkinslog, core_fil
             remote_files = OrderedDict()
             remote_files["local configuration files"] = {
                 "source": daos_test_log_dir,
-                "destination": os.path.join(self.job_results_dir, "latest", "daos_configs"),
+                "destination": os.path.join(self.job_results_dir, "latest", self.RESULTS_DIRS[0]),
```

Comment:
Minor - I actually don't like using `RESULTS_DIRS[0]` like this because it requires looking at the definition to know what the name is, and implicitly requires `RESULTS_DIRS` to stay in order

Reply:
It's not a big deal now, but if we had, say, 15 directories, it seeing `RESULTS_DIRS[15]` out of context would be confusing.

Reply:
Any idea why we didn't change it as dictionary? It would be nice to have something like RESULTS_DIRS["daos_config"]